packages/core/src/httpClient/retryValidator.ts

```diff
@@ -51,10 +51,11 @@ export const createUniversalRetryValidator =
     if (retryCount >= maxRetries) {
       return false;
     }
-    // Always retry http 429 too many requests
+    // Always retry http 429 (Too Many Requests)
     if (response.status === 429) {
       return true;
     }
+    // Bu default, retrying requests verbs meant to be idempotent
```

Comment:
```suggestion
    // By default, retry requests with HTTP verbs that are meant to be idempotent
```
I had to read the sentence several times before I got it